framework/src/modules/chain/migrations/sql/20190617105815_create_chain_meta_table.sql

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+
+/*
+  DESCRIPTION: Creates meta table for BFT.
+
+  PARAMETERS: None
+*/
+
+CREATE TABLE IF NOT EXISTS "chain_meta" (
+  "key" VARCHAR(40) UNIQUE NOT NULL PRIMARY KEY,
+  "value" TEXT NOT NULL
```

Comment:
why not JSON field here?

Reply:
Its upto usage, if you want to store serialized json or just primitive values. We don't provide feature to search in the value, so no need to store it as JSON.